### DISCUSSION TITLE
Boot

### DISCUSSION THREAD

**Discussion by manu20247** [helloSystem/hello#568] [Ideas]
Is there a possibility of putting the boot manager in the Hello System?  I found it very practical, especially for lay users.  Attached is the captured image.

![20240417_171038](https://github.com/helloSystem/hello/assets/163211417/a27847cc-efe0-41e5-9fde-e319a50dc994)

**Comment by probonopd** [comment]
rEFInd should be able to boot helloSystem, but we are currently not planning to install it by default as we don't support multiboot scenarios out of the box.
